src/navigations/HomeBottomTabs.js

```diff
@@ -193,14 +178,18 @@ function HomeBottomTabs({navigation}) {
 
   React.useEffect(() => {
     createChannel();
-    requestPermission();
     updateProfileAtomId();
 
     messaging().setBackgroundMessageHandler(async (remoteMessage) => {
       console.log('Message handled in the background!', remoteMessage);
       handlePushNotif(remoteMessage);
     });
 
+    messaging().setBackgroundMessageHandler(async (remoteMessage) => {
+      console.log('Message handled in the background!', remoteMessage);
+      handlePushNotif(remoteMessage);
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There are two identical `setBackgroundMessageHandler` calls in the `useEffect` hook. This seems redundant and could potentially lead to unexpected behavior or performance issues. Consider removing one of them.
```diff
  React.useEffect(() => {
     createChannel();
     updateProfileAtomId();

     messaging().setBackgroundMessageHandler(async (remoteMessage) => {
       console.log('Message handled in the background!', remoteMessage);
       handlePushNotif(remoteMessage);
     });

-    messaging().setBackgroundMessageHandler(async (remoteMessage) => {
-      console.log('Message handled in the background!', remoteMessage);
-      handlePushNotif(remoteMessage);
-    });
  
     const unsubscribe = messaging().onMessage((remoteMessage) => {
       // eslint-disable-next-line no-unused-expressions
       handlePushNotif(remoteMessage);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->